Database/Corrections/sodQuestFixes.lua

```diff
@@ -2928,6 +2928,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -402852,
             [questKeys.zoneOrSort] = sortKeys.PRIEST,
         },
+        [90194] = {
+            [questKeys.name] = "Between the Eyes",
+            [questKeys.startedBy] = {nil,{386777}},
```

Comment:
Same as before, this object has multiple spawns which need to be reduced to a single one.